drivers/sensor/lis2dh/lis2dh.c

```diff
@@ -430,6 +431,46 @@ int lis2dh_init(const struct device *dev)
 					LIS2DH_ODR_BITS);
 }
 
+#ifdef CONFIG_PM_DEVICE
+static int lis2dh_pm_action(const struct device *dev,
+			    enum pm_device_action action)
+{
+	int status;
+	struct lis2dh_data *lis2dh = dev->data;
+
+	switch (action) {
+	case PM_DEVICE_ACTION_RESUME:
+		/* Resume previous mode. */
+		status = lis2dh->hw_tf->write_reg(dev, LIS2DH_REG_CTRL1,
+						  lis2dh->reg_ctrl1_active_val);
+		if (status < 0) {
+			LOG_ERR("LIS2DH_REG_CTRL1 write failed: %d", status);
```

Comment:
last nitpick, for consistency:

```suggestion
			LOG_ERR("failed to write reg_crtl1");
```